### ISSUE TITLE
Images missing and large code blocks not formatted correctly in medium article

### ISSUE THREAD

**Issue by jacksonh** [omnivore-app/omnivore#594]
I had to capture this one with the extension as it was blocked in puppeteer 

https://medium.com/devtechie/swiftui-textfield-a-closer-look-2257fd3053bb

**Comment by jacksonh** [comment]
It looks like the image proxy is timing out trying to fetch this image: https://miro.medium.com/max/928/1*UYMk0hoNDKtoPP3gttN_bg.gif

**Comment by coombsj** [comment]
@jacksonh , still seeing issues with images in saved medium articles. All showing broken links.

**Comment by jacksonh** [comment]
Thanks, Medium definitely has been causing issues lately.

**Comment by jacksonh** [comment]
> @jacksonh , still seeing issues with images in saved medium articles. All showing broken links.

Do you mind providing a link that failed?

**Comment by coombsj** [comment]
Here is an example of the image link that is broken when looking at article in Omnivore:
https://proxy-prod.omnivore-image-cache.app/720x0,swRz2gvcqb0SzZ83mTKMHEZSHJpBbBZxJA7qGWe2zrcQ/https://miro.medium.com/v2/resize:fit:720/format:webp/1*KRcHGG8gGJ7s-96X2N7sPw.png

**Comment by coombsj** [comment]
Though when I click directly on that link, I am able to see it, though I am dealing with Zscaler on this machine.

**Comment by jacksonh** [comment]
yeah it looks like eventually that one was ingested, but its very slow.

**Comment by coombsj** [comment]
I can view the images just fine in the mobile app. So maybe it is the filtering due to corporate VPN that is causing the problem on this machine.

**Comment by jacksonh** [comment]
@coombsj are you able to open developer tools in your browser, and click on the network tab and see if any requests have failed? (They'd be in red if they failed), you probably have to refresh the page.

**Comment by coombsj** [comment]
Not seeing any red failed requests in the network tab.

**Comment by coombsj** [comment]
I am seeing some failures in the console though 
![image](https://github.com/omnivore-app/omnivore/assets/825918/b2c87e1e-2ea9-41b4-9cbd-f2bc22fdeec5)

**Comment by jacksonh** [comment]
ok so i think i see two things happening there:

1. the red stuff is just our analytics being blocked which is fine. 
2. the image requests i think were blocked because they didn't return images, which probably means they were taking too long to load, our proxy returned an error message (instead of the image), and then the browser blocked the request because only image requests are allowed to our proxy.

**Comment by Podginator** [comment]
I think that I fixed this issue recently, it was due to image sets. We parse the image sets and get the biggest file from it.
